mlir/lib/Target/LLVMIR/ConvertToLLVMIR.cpp

```diff
@@ -31,6 +34,13 @@ void registerToLLVMIRTranslation() {
         if (!llvmModule)
           return failure();
 
+        // When printing LLVM IR, we should convert the module to the debug info
+        // format that LLVM expects us to print.
+        // See https://llvm.org/docs/RemoveDIsDebugInfo.html
+        llvm::ScopedDbgInfoFormatSetter FormatSetter(*llvmModule,
```

Comment:
```suggestion
        llvm::ScopedDbgInfoFormatSetter formatSetter(*llvmModule,
```
ultra nit: variables should start with lowercase in mlir.